src/index.js

```diff
@@ -20,46 +18,64 @@ function response(statusCode, body) {
 }
 
 exports.handler = async (event, context, callback) => {
-    // // For keeping the browser launch
-    context.callbackWaitsForEmptyEventLoop = false;
-    const browser = await getChrome();
 
-    console.log("got browser");
+    console.log(event)
 
     if (!event.queryStringParameters) {
-        return response(400, "You need a url");
+        return response(400, "No query parameters provided");
     }
 
-    const targetUrl = event.queryStringParameters.url;
-    const code = event.queryStringParameters.code;
-    const codeType = event.queryStringParameters.codeType;
-    const urlencoded = event.queryStringParameters.urlencoded === "true";
+    const urlToScreenshot = event.queryStringParameters.url;
+    const delayMillis = event.queryStringParameters.delay || 4000;
+
+    const width = event.queryStringParameters.width || 800;
+    const height = event.queryStringParameters.height || 500;
+
+    const destination = event.queryStringParameters.destination;  // E.g. prod/4403/offer/1234/referrer_fulfill_20200102175601
```

Comment:
Doesn't actually matter, but knowledge for the future;

```suggestion
    const {
        url: urlToScreenshot,
        delay: delayMillis = 4000,
        width = 800,
        height = 500,
        destination
    } = event.queryStringParameters;
```